cypress/plugin.js

```diff
@@ -0,0 +1,5 @@
+require('..').init({
```

Comment:
these files are now easier to export, but I don't know if this scales well: a folder per plugin?

Reply:
I think it's fine since we won't need to do this often, and eventually plugins will be exported in a different way.